app/script/util/Environment.js

```diff
@@ -46,39 +46,39 @@ window.z.util = z.util || {};
   };
 
   const _check = {
-    get_version: () => {
+    getVersion: () => {
       if (window.platform.version) {
         return window.parseInt(window.platform.version.split('.')[0], 10);
       }
```

Comment:
`getVersion: () => window.platform.version ? window.parseInt(window.platform.version.split('.')[0], 10) : undefined;`

Reply:
I used `null`, otherwise it's over 120 :) Is that OK?